pinot-segment-local/src/test/java/org/apache/pinot/segment/local/indexsegment/mutable/MutableSegmentImplTestUtils.java

```diff
@@ -98,6 +98,20 @@ public static MutableSegmentImpl createMutableSegmentImpl(Schema schema, Set<Str
       UpsertConfig upsertConfig, String timeColumnName, PartitionUpsertMetadataManager partitionUpsertMetadataManager,
       DedupConfig dedupConfig, PartitionDedupMetadataManager partitionDedupMetadataManager, ServerMetrics serverMetrics,
       List<AggregationConfig> aggregationConfigs) {
+    return createMutableSegmentImpl(schema, noDictionaryColumns,
+        varLengthDictionaryColumns, invertedIndexColumns,
+        jsonIndexConfigs, aggregateMetrics, nullHandlingEnabled,
+        upsertConfig, timeColumnName, partitionUpsertMetadataManager,
+        dedupConfig, partitionDedupMetadataManager, serverMetrics,
+        aggregationConfigs, false);
+  }
+
+  public static MutableSegmentImpl createMutableSegmentImpl(Schema schema, Set<String> noDictionaryColumns,
```

Comment:
We can revert this since the flag is removed

Reply:
missed it, my bad.